producer-services/src/main/scala/za/co/absa/spline/producer/service/model/ServerStatus.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.producer.service.model
+
+import io.swagger.annotations.ApiModel
+
+
+@ApiModel(description = "Attribute Dependency Node")
+case class ServerStatus(
+  status: String,
+  version: String,
+  requestDecompression: Boolean
```

Comment:
I'm thinking about how the status JSON will look like when and if we add more flags to it.
Although any string would work, ideally we need to have some naming convention to follow.

First of all I would name a feature not after what it's doing under the hood, but rather after what I can use it for.
E.g. as a client I want to know if I can send a compressed request to a server, so for the respective feature flag name, `supportsRequestCompression` would do better than `requestDecompression` IMO.

The naming convention I'd like to propose is: `supportsXXXX: true` for features.
Another approach would be to have a `flags:[]` properties that would simply list all features enabled. E.g.
```json
{
  "version": "1.2.3",
  "flags": [ "request-compression", "foo", "bar" ]
}
```

What do you think?